src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/ExpressionBuilder.java

```diff
@@ -135,4 +135,31 @@ private static Expression<BuiltCondition> buildExpressionRecursive(
     return ExpressionUtils.buildExpression(
         conditionExpressions, logicalExpression.getLogicalRelation().getOperator());
   }
+
+  public static List<Object> getExpressionValuesInOrder(Expression<BuiltCondition> expression) {
```

Comment:
Could you add a Javadoc explaining what this does?

Reply:
sure!